projects/laji/src/app/+project-form/results/biomon-result/biomon-result-map/biomon-result-map.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Doesn't really matter, but personally I like to omit the auto generated unit tests, since we aren't running unit tests anyway this is dead code.